fgpyo/collections/__init__.py

```diff
@@ -0,0 +1,84 @@
+from typing import Any
+from typing import Callable
+from typing import Generic
+from typing import Iterable
+from typing import Iterator
+from typing import List
+from typing import TypeVar
+from typing import Union
+
+
```

Comment:
1. I think we should give attribution to where this came from
2. I think the long docs with examples are super useful to new folks on our team to show how to use this like you did for the other modules.  Can you copy those over too?

Reply:
I thought I had copied those module docs - have now.
Re: attribution I added a section to the LICENSE doc.  I think that's sufficient to merge but if you feel like we should add more elsewhere let me know.